Resources/public/js/views/cof-contenttypeselectorview.js

```diff
@@ -0,0 +1,179 @@
+                contentTypeGroups = this.get('contentTypeGroups');
+
+            contentTypeGroups.forEach(function (contentTypeGroup) {
+                contentTypeGroup.get('contentTypes').forEach(function (contentType) {
+                    if (contentTypeItemId === contentType.get('id')) {
+                        var description = contentType.get('descriptions')[contentType.get('mainLanguageCode')];
+
+                        contentTypeItem.setAttribute(ATTR_DESCRIPTION, description);
+                    }
+                });
+            });
+
+            /**
+             * Informs an item is selected.
+             * Listened in the eZS.CreateContentPopupView
+             *
+             * @param text {String} text to show on remove button
+             */
+            this.fire('itemSelected', {text: event.text});
+        },
+
+        /**
+         * Shows the tooltip.
+         *
+         * @method showTooltip
+         */
+        showTooltip: function () {
+            this._toggleTooltip(true);
+
+            this._setCorrectTooltipPositionOnScroll();
+            this._correctTooltipPosition();
+        },
+
+        /**
+         * Hides the tooltip.
+         *
+         * @method hideTooltip
+         */
+        hideTooltip: function () {
+            this._toggleTooltip(false);
+
+            if (this._scrollContentTypeHandler) {
+                this._scrollContentTypeHandler.detach();
+            }
+        },
+
+        /**
+         * Toggle visibility of the tooltip.
+         *
+         * @protected
+         * @method _toggleTooltip
+         * @param show {Boolean} should show the tooltip?
+         */
+        _toggleTooltip: function (show) {
+            var methodName = show ? 'removeClass' : 'addClass',
+                container = this.get('container'),
+                selectedItem = container.one(SELECTOR_ITEM_SELECTED),
+                tooltip = container.one(SELECTOR_TOOLTIP);
+
+            if (selectedItem) {
+                tooltip.setHTML(selectedItem.getAttribute(ATTR_DESCRIPTION));
+
+                tooltip[methodName](CLASS_HIDDEN);
+            }
+        },
+
+        /**
+         * Sets detect selected content type position handler to manage the tooltip position.
+         *
+         * @protected
+         * @method _setCorrectTooltipPositionOnScroll
+         */
+        _setCorrectTooltipPositionOnScroll: function () {
+            this._scrollContentTypeHandler = this.get('container').one(SELECTOR_CONTENT_TYPE_LIST).on('scroll', Y.bind(this._correctTooltipPosition, this));
+        },
+
+        /**
+         * Corrects the tooltip position.
+         *
+         * @protected
+         * @method _correctTooltipPosition
+         */
+        _correctTooltipPosition: function () {
+            window.clearTimeout(this._correctTooltipPositionTimeout);
+
+            this._correctTooltipPositionTimeout = window.setTimeout(Y.bind(function () {
```

Comment:
I would move this callback function into private method to make it more readable.